resource/api/swagger.yml

```diff
@@ -9,7 +9,7 @@ produces:
 schemes:
   - http
   - https
-swagger: '2.0'
+swagger: "2.0"
```

Comment:
Can we find a linter or a best practice for swagger file to avoid this kind of things ?